internal/main.go

```diff
@@ -120,6 +150,7 @@ func (e *Env) processIds(c *gin.Context) {
 
 	var wg sync.WaitGroup
 	movieCh := make(chan Movie, len(media.Ids))
+	TVCh := make(chan TV, len(media.Ids))
```

Comment:
Should probably be `tvCh` for out naming convention to make sense

Reply:
Done